src/components/reports/ReportForm.tsx

```diff
@@ -29,59 +33,84 @@ const initialValues: FormValues = {
+    reasonForTestText: "Sequence variant screening in Donald Duck because of epilepsy and atypical absences. " +
+      "An SLC2A1 variant is suspected.",
   },
   variant: {
-    gene: "",
-    genomicHGVS: "",
-    inheritanceMethod: "",
-    classification: "",
-    proteinHGVS: "",
-    referenceNucleotide: "",
-    transcript: "",
-    variantNucleotide: "",
-    zygosity: "",
-    classificationEvidence: "",
+    gene: "GNAO1",
+    genomicHGVS: "NM_020988.2:c.119G>T",
+    inheritanceMethod: "AD",
+    // will also need a code
+    classification: "Likely Pathogenic",
+    proteinHGVS: "p.(Gly40Val)",
+    referenceNucleotide: "G",
+    transcript: "NM_006516.2",
+    variantNucleotide: "T",
+    zygosity: "hetezygote",
+    classificationEvidence: "absent from the gnomAD population database (PM2_Moderate)." +
+      "affects a gene with a low tolerance for missense variation (PP2_Supporting). " +
+      "predicted to be deleterious by in silico prediction tools (PP3_Supporting). " +
+      "similar variants affecting the same amino acid c.118G>A p.(Gly40Arg), " +
+      "c.118G>C p.(Gly40Arg), c.118G>T p.(Gly40Trp) & c.119G>A p.(Gly40Glu) " +
+      "have been previously reported in the literature (1-2) (PM5_Moderate) reported on ClinVar as likely pathogenic.",
   },
   result: {
-    resultSummary: "",
-    authorisingDate: "",
-    authorisingScientist: "",
-    authorisingScientistTitle: "",
-    furtherTesting: "",
-    reportingDate: "",
-    reportingScientist: "",
-    reportingScientistTitle: "",
-    testMethodology: "",
+    resultSummary: "NAO1: Heterozygous pathogenic variants cause EIEE17 (MIM 615473) " +
+      "or neurodevelopmental disorder with involuntary movements without epilepsy (MIM 617493). " +
+      "Clinical features range from severe motor and cognitive impairment with marked choreoathetosis, " +
+      "self-injurious behaviour and epileptic encephalopathy, to a milder course with moderate developmental delay, " +
+      "complex stereotypies (facial dyskinesia) and mild epilepsy.",
+    authorisingDate: "2021-04-25",
+    authorisingScientist: "Lucy Jones",
+    authorisingScientistTitle: "Consultant Clinical Scientist",
+    furtherTesting: "Testing of Donald Duck's parents is recommended to determine whether the GNAO1 " +
+      "c.119G>T p.(Gly40Val) likely pathogenic variant has arisen de novo and to assess the recurrence risk. " +
+      "Please include clinical information for the parents. " +
+      "A referral to their local clinical genetics service may be appropriate for this family.",
+    reportingDate: "2021-04-25",
+    reportingScientist: "Ana Pietra",
+    reportingScientistTitle: "Clinical Scientist",
+    testMethodology: "Screening of 82 genes associated with severe delay and seizures " +
+      "... Variants are classified using the ACMG/AMP guidelines (Richards et al 2015 Genet Med) " +
+      "/ACGS Best Practice guidelines (2019).",
   }
 };
 
-
 const ReportForm = () => {
```

Comment:
Have you not assigned a type here because the component takes no arguments?

Reply:
Yep, though if you think it's better to define I'm happy to

Reply:
It's an interesting one. It comes back to the discussion of whether absence indicates intention. The lack of type here implicitly suggests we do not expect to receive props, but should we be explicit about this? I tried a few implementations but it caused some errors. I'd leave it as it is to be honest 😛